frontend/src/bundles/common/enums/app-route.enum.ts

```diff
@@ -2,8 +2,17 @@ const AppRoute = {
     ROOT: '/',
     SIGN_IN: '/sign-in',
     SIGN_UP: '/sign-up',
+    USER: '/user',
     DASHBOARD: '/dashboard',
     UI: '/ui',
+    PRICING: '/pricing',
+    BANKS: '/banks',
+    HELP: '/help',
+    ABOUT: '/about',
+    BLOG: '/blog',
+    CONTACT: '/contact',
```

Comment:
I am not sure that we need these pages
Check our spec

Reply:
it was merge from [`task/ss-73-add-header`](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/tree/task/ss-73-add-header) branch